mrp_lock_lot/models/mrp_production_lot.py

```diff
@@ -21,7 +21,14 @@ class StockProductionLot(models.Model):
 
     @api.one
     def _get_locked_value(self):
-        return self.product_id.categ_id.lot_default_locked
+        '''Lock by default?'''
+        categ = self.product_id.categ_id
```

Comment:
Won't it be confusing to have 2 checkboxes? maybe we should propagate changes from parent to children instead like in https://github.com/numerigraphe/numerigraphe-addons/blob/7.0/stock_prodlot_lock/product.py#L40 ?